app/data/JsonConversions.scala

```diff
@@ -2,35 +2,78 @@ package data
 
 import com.gu.contentatom.thrift._
 import com.gu.contentatom.thrift.atom.media._
-import play.api.libs.json._
 import play.api.libs.functional.syntax._
+import play.api.libs.json._
 
 object JsonConversions {
 
   implicit val atomType = Writes[AtomType](at => JsString(at.name.toLowerCase))
 
+  implicit val category = Writes[Category](category => JsString(category.name.toLowerCase))
+
   implicit val mediaAsset = (
     (__ \ "id").write[String] and
-      (__ \ "version").write[Long] and
-      (__ \ "platform").write[String] and
-      (__ \ "assetType").write[String]
+    (__ \ "version").write[Long] and
+    (__ \ "platform").write[String] and
+    (__ \ "assetType").write[String] and
+    (__ \ "mimeType").writeNullable[String]
   ) { asset: Asset =>
-    asset match { case Asset(assetType, version, id, platform) => (id, version, platform.name, assetType.name) }
+    asset match {
+      case Asset(assetType, version, id, platform, mimeType) => (id, version, platform.name, assetType.name, mimeType)
+    }
   }
 
   implicit val atomDataMedia = (
     (__ \ "assets").write[Seq[Asset]] and
-      (__ \ "activeVersion").write[Long]
-  ) { mediaAtom: MediaAtom =>
-    (mediaAtom.assets, mediaAtom.activeVersion)
+    (__ \ "activeVersion").writeNullable[Long] and
+    (__ \ "title").write[String] and
+    (__ \ "category").write[Category] and
+    (__ \ "plutoProjectId").writeNullable[String] and
```

Comment:
@paulmr Can you answer this one?  It looks like the prop was added here: https://github.com/guardian/content-atom/commit/21247f54c677d5506e238fa2046917af2822f2c4